src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -50,6 +59,16 @@ public class WwwAuthenticateParameters
         /// </summary>
         public string Error { get; set; }
 
+        /// <summary>
+        /// Scheme.
+        /// </summary>
+        public string Scheme { get; set; } 
```

Comment:
I would rename it `AuthenticationProtocol`. Scheme is overloaded (with ASP.NET Core authentication scheme)

Reply:
The spec mentions "auth-scheme" but I agree it's overloaded

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate#syntax

Reply:
If the spec mentiosn auth-scheme, let's use AuthScheme, but let's make sure the comment provides a link to the spec, and has the name protocol.